samples/net/sockets/big_http_download/src/big_http_download.c

```diff
@@ -4,6 +4,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <logging/log.h>
```

Comment:
big_http_download is a POSIX-compatible sample (i.e. written with purely POSIX API, and portable to any POSIX system). Please run `make -f Makefile.posix` in the sample dir to ensure it remains so. (This change likely breaks that.)